plugins/elevenlabs/client/src/index.ts

```diff
@@ -0,0 +1,55 @@
+// DOCUMENTED 
+/**
+ * A plugin for interacting with elevenlabs's API.
+ * @class
+ */
+import { ClientPlugin, InputControl } from '@magickml/core'
+import shared from '@magickml/plugin-elevenlabs-shared'
+
+// Importing shared variables from plugin-elevenlabs-shared module
+const { secrets, completionProviders } = shared
+
+// Input controls for textToSpeech completion
+const textToSpeechControls = [
+  {
+    type: InputControl,
+    dataKey: 'stability',
+    name: 'Stability (0-1.0)',
+    icon: 'moon',
+    defaultValue: 0,
+  },
+  {
+    type: InputControl,
+    dataKey: 'similarity_boost',
+    name: 'Similarity Boost (0-1.0)',
+    icon: 'moon',
+    defaultValue: 0,
+  },
+  {
+    type: InputControl,
+    dataKey: 'voice_id',
+    name: 'Voice ID',
+    icon: 'moon',
+    defaultValue: '',
+  },
+];
+
+// Object containing all input controls for different completion types
```

Comment:
Last time mentioning sentence case (preferring) versus other.

Reply:
im just matching the existing plugin formats

Reply:
Will circle back later.